src/client/components/SocialButtons.tsx

```diff
@@ -18,16 +18,23 @@ const containerStyles = css`
     flex-direction: row;
   }
   justify-content: center;
-  margin-top: ${space[12]}px;
-  margin-bottom: ${space[12]}px;
+  margin-top: ${space[5]}px;
+  ${from.mobileMedium} {
+    margin-top: ${space[6]}px;
+  }
+  margin-bottom: 60px;
+  ${from.desktop} {
+    margin-bottom: ${space[24]}px;
```

Comment:
Just thinking, if we come to use the social buttons in the future outside of this context, would the large margin here be desirable.

Thoughts on adding the margins to a wrapper instead?

Reply:
I can't see a future scenario where these buttons would be used outside the context of sign-in/registration. If there is a future use case for these buttons perhaps it can be refactored at the time